jooby/src/test/java/org/jooby/RequestLoggerTest.java

```diff
@@ -147,6 +169,13 @@ private Block path(final String path) {
     };
   }
 
+  private Block query(final String query) {
+    return unit -> {
+      Request req = unit.get(Request.class);
+      expect(req.queryString()).andReturn(Optional.of(query)).atLeastOnce();
```

Comment:
I'm not sure I like it that I'm the only one that uses .atLeastOnce() in the test
Any better way? (The RequestLogger call several times to queryString())